datalad/tests/utils.py

```diff
@@ -549,7 +550,7 @@ def newfunc(*args, **kwargs):
                                  "CWD changed from %s to %s" % (cwd_before, cwd_after))
 
         if exc_info is not None:
-            raise exc_info[0](exc_info[1], exc_info[2])
+            reraise(*exc_info)
```

Comment:
holy crap -- really, that was that simple? ;)  we have just merged by fix/workaround, so I expect this change/merge to fail.  but mine works too (and with some additional test)